docs/deploy/confluent-cloud.md

```diff
@@ -0,0 +1,119 @@
+SPRING_KAFKA_PROPERTIES_SASL_MECHANISM=PLAIN
+SPRING_KAFKA_PROPERTIES_CLIENT_DNS_LOOKUP=use_all_dns_ips
+SPRING_KAFKA_PROPERTIES_BASIC_AUTH_CREDENTIALS_SOURCE=USER_INFO
+SPRING_KAFKA_PROPERTIES_BASIC_AUTH_USER_INFO=U4UXEN6SP3KEWMO4:ESju8ZfETDm2RZo/6KO057hPYs2TGjFKmvMWUFnlJ3uKubFbB1Sfs7aOjUVT6CGe
+```
+
+Next, configure datahub-frontend to connect to Confluent Cloud by changing `docker/datahub-frontend/env/docker.env`:
+
+```
+KAFKA_BOOTSTRAP_SERVER=pkc-e8mp5.eu-west-1.aws.confluent.cloud:9092
+
+# Confluent Cloud Configs
+KAFKA_PROPERTIES_SECURITY_PROTOCOL=SASL_SSL
+KAFKA_PROPERTIES_SASL_JAAS_CONFIG=org.apache.kafka.common.security.plain.PlainLoginModule   required username='XFA35EL7QFUQP2PA' password='zmkfP6Envr9TYutsjLB3ZYfrk+yfCXD8sQHCE3EMp57A2jNs4RR7J1bU9k6lM6rU';
+KAFKA_PROPERTIES_SASL_MECHANISM=PLAIN
+KAFKA_PROPERTIES_CLIENT_DNS_LOOKUP=use_all_dns_ips
+KAFKA_PROPERTIES_BASIC_AUTH_CREDENTIALS_SOURCE=USER_INFO
+KAFKA_PROPERTIES_BASIC_AUTH_USER_INFO=U4UXEN6SP3KEWMO4:ESju8ZfETDm2RZo/6KO057hPYs2TGjFKmvMWUFnlJ3uKubFbB1Sfs7aOjUVT6CGe
+```
+
+Note that this step is only required if DATAHUB_ANALYTICS_ENABLED is not set to false.
+
+If you're deploying with Docker Compose, you do not need to deploy the Zookeeper, Kafka Broker, or Schema Registry containers that ship by default.
+
+### Helm
+
+If you're deploying to K8s using Helm, you can simply change the `datahub-helm` values.yml to point to Confluent Cloud and disable some default containers:
+
+First, disable the `cp-schema-registry` service:
+
+```
+cp-schema-registry:
+  enabled: false 
+```
+
+Next, disable the `kafkaSetupJob` service:
+
+```
+kafkaSetupJob:
+    enabled: false
+```
+
+Then, update the `kafka` configurations to point to your Confluent Cloud broker and schema registry instance, along with the topics you've created in Step 1:
+
+```
+kafka:
+      bootstrap:
+        server: pkc-e8mp5.eu-west-1.aws.confluent.cloud:9092
+      schemaregistry:
+        url: https://psrc-yorrp.us-east-2.aws.confluent.cloud
+```
+
+Finally, set the required Confluent Kafka properties under the `springKafkaConfigurationOverrides` block:
+
+```
+springKafkaConfigurationOverrides:
+      security.protocol: SASL
+      sasl.jaas.config: org.apache.kafka.common.security.plain.PlainLoginModule   required username='XFA35EL7QFUQP2PA' password='zmkfP6Envr9TYutsjLB3ZYfrk+yfCXD8sQHCE3EMp57A2jNs4RR7J1bU9k6lM6rU'; # Provided inside Confluent Control Center
+      sasl.mechanism: PLAIN
+      client.dns.lookup: use_all_dns_ips
+      basic.auth.credentials.source: USER_INFO
+      basic.auth.user.info: U4UXEN6SP3KEWMO4:ESju8ZfETDm2RZo/6KO057hPYs2TGjFKmvMWUFnlJ3uKubFbB1Sfs7aOjUVT6CGe # Provided within Confluent Control Center
+```
+
+Typically, we suggest that all secrets are managed using K8s native secret manager. If you are using this, you can replace actual credentials with references to your secrets. For example,
```

Comment:
I don't think the way it is given will use K8s native secret manager. What I had given was a ansible template. So these were being replaced by ansible.

Reply:
yes - good catch. updating the doc